spec/gitlab/client/groups_spec.rb

```diff
@@ -37,6 +37,25 @@
     end
   end
 
+  describe ".create_group_with_description" do
```

Comment:
Can you make this a context under the already existing `describe ".create_group"` block?